streampark-common/src/main/java/org/apache/streampark/common/enums/RestoreMode.java

```diff
@@ -21,8 +21,14 @@
 import java.util.Arrays;
 
 public enum RestoreMode implements Serializable {
-  NO_CLAIM(1),
-  CLAIM(2),
+
+  /** */
+  CLAIM(1),
+
+  /** */
+  NO_CLAIM(2),
+
+  /** */
   LEGACY(3);
 
```

Comment:
How about `private final int value;`->`private final int mode;` ?

Reply:
Sounds good. I would like to ask what these values in this class mean like ’LEGACY‘，“NO_CLAIM”, and I want to perfect their annotations

Reply:
> Sounds good. I would like to ask what these values in this class mean like ’LEGACY‘，“NO_CLAIM”, and I want to perfect their annotations

https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/ops/state/savepoints/